Makefile

```diff
@@ -9,6 +9,7 @@ IOS_VERSION ?= 11.3
 IPHONE_NAME ?= iPhone 8
 BRANCH ?= master
 DIST_BRANCH = $(RELEASE)-dist
+OPENTOK_VERSION = 2.10.2
```

Comment:
Could we change this to `OPENTOK_VERSION ?= 2.10.2` That way the value that we pass through as an environment variable will be used if it's set.